zerver/views/__init__.py

```diff
@@ -827,6 +834,8 @@ def home(request):
         avatar_url            = avatar_url(user_profile),
         mandatory_topics      = user_profile.realm.mandatory_topics,
         show_digest_email     = user_profile.realm.show_digest_email,
+
+        should_offer_mobile_redirect = suggestMobileForward,
```

Comment:
Probably should rename both of these variables to something clearer like `offer_mobile_redirect`